loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -594,6 +596,70 @@ module BABYLON.GLTF2 {
             var material = getMaterial(runtime, primitive.material);
             multiMat.subMaterials.push(material);
 
+            // Morph Targets
+            if (primitive.targets !== undefined) {
+                morphTargetManager = new BABYLON.MorphTargetManager();
+
+                for (var index2 = 0; index2 < primitive.targets.length; index2++) {
+                    var target = primitive.targets[index2];
+
+                    var weight = 0.0;
+                    if (node.weights !== undefined) {
+                        weight = node.weights[index2];
+                    }
+                    else if (mesh.weights !== undefined) {
+                        weight = mesh.weights[index2];
+                    }
+
+                    var morph = new BABYLON.MorphTarget("morph" + index2, weight);

```

Comment:
It might be worthwhile to _try_ and get a reasonable name for the morph target from the file.  Right now, the only way to do that would be to grab the (optional) `name` from the `accessor` of one of the three `semantic`s.  Using a generated name like this seems like a reasonable fallback.  Can the name be changed after the `BABYLON.MorphTarget` has been created?  If not, then you might need to _set_ the data on `morph` outside of the loop.